snippets/firestore/firestore_async.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+from firebase_admin import firestore_async
+
+# pylint: disable=invalid-name
+def init_firestore_client():
+    # [START init_firestore_client]
```

Comment:
Do we want to mark this as async? `init_firestore_async_client():` and `# [START init_firestore_async_client]`?

Reply:
Done